app/src/main/java/org/openobservatory/ooniprobe/model/NetworkMeasurement.java

```diff
@@ -28,6 +29,7 @@ public NetworkMeasurement(String name) {
         this.anomaly = 0;
     }
 
+    @NonNull
```

Comment:
Here you are basically hinting (or asserting?) that the return value is `non-null`?

Reply:
Don't know, just copied from a working example

Reply:
https://en.wikipedia.org/wiki/Cargo_cult_programming